src/styles/Components/src/tokens-menubutton.css

```diff
@@ -0,0 +1,20 @@
+:root {
+    --denhaag-menubutton-active-color: var(--denhaag-menubutton-hover-color);
+    --denhaag-menubutton-active-font-weight: 700;
+
+    --denhaag-menubutton-chevron-active-color: var(--denhaag-menubutton-hover-color);
+    --denhaag-menubutton-chevron-font-size: 12px;
+    --denhaag-menubutton-chevron-hover-color: var(--denhaag-menubutton-chevron--active-color);
+    --denhaag-menubutton-chevron-padding-block-start: 2px;
+    --denhaag-menubutton-chevron-padding-inline-start: 4px;
+
+    --denhaag-menubutton-color: var(--denhaag-grey-4);
+
+    --denhaag-menubutton-font-family: var(--denhaag-font-family);
+    --denhaag-menubutton-font-size: var(--denhaag-fontsize-medium);
+    --denhaag-menubutton-font-weight: 500;
+
+    --denhaag-menubutton-hover-color: var(--denhaag-blue-3);
+    --denhaag-menubutton-line-height: 21px;
+    --denhaag-menubutton-margin-inline-end: 40px;
+}
```

Comment:
The ordering of the tokens feels a little awkward. Some token values refer to a token that's defined after itself rather than before itself.

Reply:
I have changed the ordering, thanks for the feedback